src/webln/NostrWeblnProvider.ts

```diff
@@ -175,6 +182,84 @@ export class NostrWebLNProvider implements WebLNProvider, Nip07Provider {
     }
   }
 
+  // TODO: refactor code in getBalance and sendPayment
```

Comment:
@im-adithya can you address the TODO?

Reply:
@im-adithya I extracted the method in https://github.com/getAlby/js-sdk/pull/45

If you think it looks good we could use that method in this PR